### DISCUSSION TITLE
What kind of API token templates should I use?

### DISCUSSION THREAD

**Discussion by francis-jjk** [kriasoft/react-starter-kit#2018] [Q&A]
For test purpose, I just wanna try if the demo project cloud be deployed.

![image](https://user-images.githubusercontent.com/110971964/200275461-bf6f98f7-11d8-4037-906a-4eaba931e867.png)

**Comment by koistya** [comment]
You can start with "Edit Cloudflare Workers" template, and provide which account and/or zone the API key would be allowed to access. When you hit `yarn deploy`, it will deploy the app to Cloudflare Workers (the `edge` package), and upload static files (the `app` package` to Cloudflare Workers KV storage).
